app/src/main/java/uk/gov/onelogin/tokens/usecases/IsAccessTokenExpired.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.onelogin.tokens.usecases
+
+import javax.inject.Inject
+
+interface IsAccessTokenExpired {
+    operator fun invoke(): Boolean
+}
+
+class IsAccessTokenExpiredImpl @Inject constructor(
+    private val getTokenExpiry: GetTokenExpiry
+) : IsAccessTokenExpired {
+    override fun invoke() =
+        getTokenExpiry()?.let { it < System.currentTimeMillis() } ?: true
```

Comment:
[NullableBooleanCheck](https://detekt.dev/docs/rules/style#nullablebooleancheck)

Not a big deal